fenix/app/src/main/res/values/strings.xml

```diff
@@ -72,7 +72,7 @@
         private mode in our new Total Private Browsing mode.
         The first parameter is the name of the app defined in app_name (for example: Firefox Nightly)
         The second parameter is the clickable link text in felt_privacy_info_card_subtitle_link_text -->
-    <string name="felt_privacy_info_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private windows. %2$s</string>
+    <string name="felt_privacy_info_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private tabs. %2$s</string>
```

Comment:
This string was added in 119 [(Bug 1847819)](https://bugzilla.mozilla.org/show_bug.cgi?id=1847819). We can't simply change the string directly anymore. We need to follow the process of adding a new string ID and marking the old one for removal if it's unused. See https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/docs/Working-with-Strings.md

Reply:
Ah I see, I created a new string and relinked the info card text to that new string.  Marked the old one for removal as well.